### PR TITLE
clean up out put of the script and add more instruction

### PR DESCRIPTION
**Pull request by ticoann**
Test with old script and compare the results. Just update the some prints statement for comparison.

### CODE REVIEW

**Comment by ticoann** [comment]
@amaltaro, Alan please review

File: bin/fix-dbs-parentage [hunk 1]
```diff
@@ -58,29 +62,29 @@ def getMissingParentSummaryUsingBlock(datasetList, fileFix=False, insertFlag=Fal
...
-                print("Aborting the procedure, Nothing updated")
-                exit(1)
-        else:
-            print("Only testing perfomance: No insert")
-            failedBlocks = dbsService.insertMissingParentageForAllFiles(dsName,  filterFilesWithParents=fileFix,
+        if fileFix:
+            failedBlocks = dbsService.insertMissingParentageForAllFiles(dsName, filterFilesWithParents=fileFix,
                                                                         insertFlag=insertFlag)
+        else:
+            failedBlocks = dbsService.fixMissingParentageDatasets(dsName, insertFlag=insertFlag)
```
Comment (amaltaro) [inline bin/fix-dbs-parentage]: Seangchan, are these two methods `insertMissingParentageForAllFiles` and `fixMissingParentageDatasets` actually writing to DBS? If so, then I'd suggest to move then somewhere else or rename this `getMissingParentSummaryUsingBlock` method name, such that it's clear what's only reading and what is writing to DBS.

File: bin/fix-dbs-parentage [hunk 2]
```diff
@@ -58,29 +62,29 @@ def getMissingParentSummaryUsingBlock(datasetList, fileFix=False, insertFlag=Fal
 
     if insertFlag:
```
Comment (amaltaro) [inline bin/fix-dbs-parentage]: If one doesn't pass `insertFlag`, it will run this method all the way until the insert and fix methods below.

**Review by amaltaro — CHANGES_REQUESTED** [review]
I made a couple of comments in the code.

File: bin/fix-dbs-parentage (hunk 1 above)
Comment (ticoann) [reply bin/fix-dbs-parentage]: It only insert to DBS when insertFlag is set to True. I can update the name.

File: bin/fix-dbs-parentage (hunk 2 above)
Comment (ticoann) [reply bin/fix-dbs-parentage]: Yes but with insertFlag = False. which run the those method without actually inserting to dbs. This is just a test case to check how long it takes and correctness of the parentage.

File: bin/fix-dbs-parentage (hunk 1 above)
Comment (amaltaro) [reply bin/fix-dbs-parentage]: Seangchan, I'd suggest then to update this method getMissingParentSummaryUsingBlock to updateParentageInfo or something like that.

Comment (ticoann) [reply bin/fix-dbs-parentage]: I will do that. Thanks,

**Comment by amaltaro** [comment]
Seangchan, is there any difference between those 2 scripts?

**Comment by ticoann** [comment]
One script is for exact same functionality for reqmgr thread (checking by announced, archived data). 
Sorry I wasn't plan to include it here, since it need some refinement and I just created for the test. I will remove it.

**Comment by amaltaro** [comment]
Seangchan, don't forget removing the script you said you didn't mean to add to this PR please. Otherwise changes look good (just make sure that script is executable, if you're planning to run it with $manage)

**Review by amaltaro — APPROVED** [review]

**Pull request merged**